distributed/cli/dask_worker.py

```diff
@@ -26,214 +26,237 @@
 
 logger = logging.getLogger("distributed.dask_worker")
 
-
 pem_file_option_type = click.Path(exists=True, resolve_path=True)
 
+worker_options = [
+    click.Option(
+        ["--tls-ca-file"],
+        type=pem_file_option_type,
+        default=None,
+        help="CA cert(s) file for TLS (in PEM format)",
+    ),
+    click.Option(
+        ["--tls-cert"],
+        type=pem_file_option_type,
+        default=None,
+        help="certificate file for TLS (in PEM format)",
+    ),
+    click.Option(
+        ["--tls-key"],
+        type=pem_file_option_type,
+        default=None,
+        help="private key file for TLS (in PEM format)",
+    ),
+    click.Option(
+        ["--worker-port"],
+        default=None,
+        help="Serving computation port, defaults to random. "
+        "When creating multiple workers with --nprocs, a sequential range of "
+        "worker ports may be used by specifying the first and last available "
+        "ports like <first-port>:<last-port>. For example, --worker-port=3000:3026 "
+        "will use ports 3000, 3001, ..., 3025, 3026.",
+    ),
+    click.Option(
+        ["--nanny-port"],
+        default=None,
+        help="Serving nanny port, defaults to random. "
+        "When creating multiple nannies with --nprocs, a sequential range of "
+        "nanny ports may be used by specifying the first and last available "
+        "ports like <first-port>:<last-port>. For example, --nanny-port=3000:3026 "
+        "will use ports 3000, 3001, ..., 3025, 3026.",
+    ),
+    click.Option(
+        ["--bokeh-port"],
+        type=int,
+        default=None,
+        help="Deprecated.  See --dashboard-address",
+    ),
+    click.Option(
+        ["--dashboard-address"],
+        type=str,
+        default=":0",
+        help="Address on which to listen for diagnostics dashboard",
+    ),
+    click.Option(
+        ["--dashboard/--no-dashboard"],
+        default=True,
+        required=False,
+        help="Launch the Dashboard [default: --dashboard]",
+    ),
+    click.Option(
+        ["--bokeh/--no-bokeh"],
+        "bokeh",
+        default=None,
+        help="Deprecated.  See --dashboard/--no-dashboard.",
+        required=False,
+    ),
+    click.Option(
+        ["--listen-address"],
+        type=str,
+        default=None,
+        help="The address to which the worker binds. Example: tcp://0.0.0.0:9000",
+    ),
+    click.Option(
+        ["--contact-address"],
+        type=str,
+        default=None,
+        help="The address the worker advertises to the scheduler for "
+        "communication with it and other workers. "
+        "Example: tcp://127.0.0.1:9000",
+    ),
+    click.Option(
+        ["--host"],
```

Comment:
Not all worker arguments make sense to add to dask-ssh (where they're used as a single flag to provide to all workers). `--host` for example is likely to differ between workers. Same with `--name` below, and likely a few others.

Reply:
Added `"pid_file", "host", "name"` in the `ignored_options` list. 
Any others that I missed?